gdpr/readme.md

```diff
@@ -0,0 +1,35 @@
+# how-to
+
+## required services and subscriptions
+
+services:
+- Audit Log Service
+- SAP HANA Cloud
+- SAP HANA Schemas & HDI Containers
+- Application Logging Service
+- Personal Data Manager
+- Authorization and Trust Management Service
+
+subscriptions:
+- Audit Log Viewer Service
+- Personal Data Manager
+
+## deploy
+
+after adding the necessary entitlements, do:
+- `cf l` to log into the respective account
+- `cd gdpr` (if still in root of `cloud-cap-samples`)
+- `npm run deploy`, which executes build and deployment via `.etc/deploy.sh`
+
+## authorization
+
+create roles for Audit Log Viewer Service and Personal Data Manager, and assign the roles to the respective users
+
+# open issues
+
+- deploy via mta, which can bind with parameters, and get rid of scripts in `.etc`
+- use approuter to remove hacky custom auth impl (`srv/auth.js`)
```

Comment:
I think these deployment todos should be solved.  Otherwise we end up w/ a lot of special deployment stuff that deviates from our documentation.

Reply:
follow-up of revised deployment guide